packages/synapse-react-client/src/style/components/_governance-dashboard.scss

```diff
@@ -26,6 +26,10 @@
   }
   .TabContentContainer {
     margin: 20px 30px;
+    .FullWidthAlert {
+      margin: -20px -30px 20px -30px;
+      width: auto;
+    }
```

Comment:
Adjust OrientationBanner positioning to be directly beneath the Submissions tab and to extend the full width of the page, rather than using the TabContentContainer margin

Reply:
Good call! Will update in the next commit